src_erl/Communication_Layer/http_Nerlserver/src/utilities_handler.erl

```diff
@@ -0,0 +1,39 @@
+-module(utilities_handler).
+
+-define(DATA_IDX , 2).
+
+-export([init/2]).
+
+init(Req0 , [MainServerPid]) ->
+    %handler for a tool connection req
+    {_,Body,_} = cowboy_req:read_body(Req0),
+    [UtilityName , IP , Port] = binary_to_term(Body),
+    case UtilityName of
+        nerlMonitor ->   
+           Graph = gen_server:call(MainServerPid , getGraph),
+           WorkersMap = ets:lookup_element(nerlnet_data , workers , ?DATA_IDX),
+           WorkersClients = maps:to_list(WorkersMap),
+           Workers = lists:flatten([atom_to_list(X)++"-"++atom_to_list(Y)++"!" || {X , Y} <- WorkersClients]),
```

Comment:
What is the role of "-" and "!" ?
"-" is very very risky it can be a part of worker or client name. 
Please explain what this line does and change marks to some safer patern e.g. @#@